Fixed button not visible completely #633

Ramreiso opened the pull request:
Signed-off-by: Ramreiso Kashung ramreisog24@gmail.com

**Description**

This PR fixes #633

**Notes for Reviewers**


**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ #633 ] Yes, I signed my commits.
 

<!--
Thank you for contributing to Meshery! 
![Screenshot from 2021-12-14 10-49-36](https://user-images.githubusercontent.com/76048154/145937945-741287cb-06cb-4fe1-b1eb-da89ac9613d3.png)


Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->
] Yes, I signed my commits.
 

<!--
Thank you for contributing to Meshery! 
![Screenshot from 2021-12-14 10-49-36](https://user-images.githubusercontent.com/76048154/145937945-741287cb-06cb-4fe1-b1eb-da89ac9613d3.png)


Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->
] Yes, I signed my commits.
 

<!--
Thank you for contributing to Meshery! 
![Screenshot from 2021-12-14 10-49-36](https://user-images.githubusercontent.com/76048154/145937945-741287cb-06cb-4fe1-b1eb-da89ac9613d3.png)


Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->
![Screenshot from 2021-12-14 10-49-36](https://user-images.githubusercontent.com/76048154/145938343-4718109b-598d-4072-b663-75dc95e74863.png)

warunicorn19 reviewed (changes requested):
Could you please get the text to be vertically centred in the button. Also, while you're in there, I just noticed that the spelling of "calendar" is wrong in the dropdown, could you please fix that too? 
<img width="1440" alt="Screenshot 2021-12-14 at 5 44 19 AM" src="https://user-images.githubusercontent.com/53532328/145983504-687580f1-1ada-4edc-9cb3-9dd529c8904e.png">

YashKamboj reviewed (changes requested):
@Ramreiso  Instead of changing the padding , why don't you justify the height of the input , and use "rem"

Ramreiso commented:
@warunicorn19 @YashKamboj 

I have updated the PR in response to changes requested.
Additionally, I have also fixed another bug where 'Participate' button was not visible completely at https://meshery.io/programs/gsoc/2020 .

Further requests or suggestions are most welcome!

YashKamboj commented:
I think this is ready to merge

YashKamboj commented:
I think this is ready to merge

YashKamboj commented:
I think this is ready to merge

YashKamboj commented:
I think this is ready to merge

shubham-tyagi commented on calendar.html:
```diff
@@ -19,7 +19,7 @@ <h2>Engage in the Community</h2>
     <h3 style="text-align:center;color:aliceblue;font-size: 1.6rem;">
-      <input style="padding-bottom:.9rem;padding-top:.9rem; margin:auto; width:80%;" type="submit" value="Join the Community">
+      <input style="padding-bottom:3rem; padding-top:0.9rem; margin:auto; width:80%;" type="submit" value="Join the Community">
     </h3>
```
Everything looks good, but maybe if we can avoid inline styling that would be great.

shubham-tyagi replied:
what do you think about this @warunicorn19?

YashKamboj replied:
@shubham-tyagi Most files have inline styling .

warunicorn19 replied:
Yeah @YashKamboj is right, generally we don't want to do inline styling, but here, since it's already there we'll let it be
